modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -142,7 +134,7 @@ definition {
 			}
 		}
 
-		task ("Verify only critical elements are highlighted") {
+		task ("Assert only critical elements are highlighted") {
 			var i = "1";
 
 			while (IsElementPresent(key_text = "${i}", locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_COUNT")) {
```

Comment:
I'd also remove the while loop and make this more clear. We're only asserting 2 known critical violations and assert that a 3rd critical violation is not present. If there's a 3rd critical element then we should probably have the test fail.